<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transferDialog</class>
 <widget class="QDialog" name="transferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sourceCard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>destCard</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceCard_lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="destCard_lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="recipientNameLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pin_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verify_pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>verify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw_pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>withdraw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
